bootstrap/bootstrap.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+############################################################
+# Help                                                     #
+############################################################
+Help()
+{
+   # Display Help
+   echo "Bootstrap spack-stack with Miniconda installation."
+   echo
+   echo "options:"
+   echo "-p     spack-stack installation prefix"
```

Comment:
Should we follow the conventions and use `--prefix` instead/in addition?

Reply:
Sure

Reply:
getopts does not support long args. I would just leave it as `-p`. There's only a single argument.

Reply:
Ok